beets/ui/commands.py

```diff
@@ -1479,17 +1479,20 @@ def move_items(lib, dest, query, copy, album, pretend, confirm=False):
             show_path_changes([(obj.path, obj.destination(basedir=dest))
                                for obj in objs])
     else:
-        if confirm:
-            objs = ui.input_select_objects(
-                u'Really %s' % act, objs,
-                lambda o: show_path_changes(
-                    [(o.path, o.destination(basedir=dest))]))
+        if export:
+            util.copy(item.path, item.destination(basedir=dest))
```

Comment:
It might be helpful to a little homework about how to define Python functions. There's lots of advice out there if you google "Python optional parameters". For example, these SO questions might help:
http://stackoverflow.com/q/43420535
http://stackoverflow.com/q/9539921